test/image-diff_test.js

```diff
@@ -45,12 +45,51 @@ describe('image-diff', function () {
     });
   });
 
+  describe('diffing different images using lower threshold', function () {
+    runImageDiff({
+      actualImage: __dirname + '/test-files/checkerboard.png',
+      expectedImage: __dirname + '/test-files/white.png',
+      diffImage: __dirname + '/actual-files/different.png',
```

Comment:
`different` is very ambiguous. Can we name the file `threshold-under` so it is semantic to the test?